.mailmap

```diff
@@ -498,3 +498,4 @@ Zhi Zhang <willzzhang@tencent.com>
 Zhi Zhang <willzzhang@tencent.com> <zhangz.david@outlook.com>
 Zhi Zhang <willzzhang@tencent.com> <zhangzhi@localhost.localdomain>
 Zhuang Xiaochun <zhuangxc89@163.com>
+Zhang Lei <243290414@qq.com> <zhanglei@trendytech.com.cn> 
```

Comment:
@theanalyst Then .organizationmap should be updated too, as per the new change. Please compare the other entries (or your own entry) between .mailmap and .organizationmap.